app/models/user.rb

```diff
@@ -342,14 +342,29 @@ def destroy
     super
   end
 
-  %w[veteran_status military_information payment].each do |emis_method|
-    define_method(emis_method) do
-      emis_model = instance_variable_get(:"@#{emis_method}")
-      return emis_model if emis_model.present?
-
-      emis_model = "EMISRedis::#{emis_method.camelize}".constantize.for_user(self)
-      instance_variable_set(:"@#{emis_method}", emis_model)
-      emis_model
+  military_information_method = 'military_information'
+  define_method(military_information_method) do
```

Comment:
the meta programming isn't really doing anything anymore here, let's just define the method directly: `def military_informatino`

Reply:
Totally on-board to delete meta programming @bosawt, but to keep the blast radius of this PR as small as possible, @rjohnson2011 can you put this method back to how it was, except remove the `veteran_status` from the first line? And then in my follow-up PR, I'm handling military_information so I'll get rid of the meta programming. I know this was leftover from my branch, so my bad for not cleaning it up!

Reply:
+ to all of this!